src/frontend/Komponenter/Behandling/Vurdering/Vurdering.tsx

```diff
@@ -205,7 +205,10 @@ export const Vurdering: React.FC<{ behandling: Behandling }> = ({ behandling })
                                         <FritekstFeltWrapper>
                                             <EnsligTextArea
                                                 label="Innstilling til NAV Klageinstans (kommer med i brev til bruker)"
-                                                value={oppdatertVurdering.innstillingKlageinstans}
+                                                value={hentInnstillingKlageinstansTekstForVurdering(
+                                                    oppdatertVurdering,
+                                                    behandling.fagsystem
+                                                )}
```

Comment:
Ble litt nysgjerrig på hva som skjer i lesevisning, for antar at vi ikke ønsker at feltet skal fylles ut med dette hvis det er lesevisning (men da er kanskje aldri denne boksen tom?). Ser at EnsligTextArea har readonly alltid satt til false og ble litt nysgjerrig på hvorfor. Kommer man aldri inn til Vurdering-komponenten hvis det er readonly? 🤔

Reply:
Det vises ikke i lesevisning, på 164 tar man en sjekk på om behandlingen er redigerbar. 
Hvis ikke så vises det en annen komponent, `VurderingLesemodus` 🙌